vertical-pod-autoscaler/pkg/recommender/logic/recommender_test.go

```diff
@@ -25,7 +25,7 @@ import (
 func TestMinResourcesApplied(t *testing.T) {
 	constEstimator := NewConstEstimator(model.Resources{
 		model.ResourceCPU:    model.CPUAmountFromCores(0.001),
-		model.ResourceMemory: model.MemoryAmountFromBytes(1e6),
```

Comment:
why do we need to change this? isn't it related to what `podMinMemoryMb` we set in tests?

Reply:
Previously, it was testing that if the estimator's recommendation was < 250MB, then the final recommendation would be set to 250MB. So since 1e6 bytes < 250MB, the later assertion would check that it's set to 250MB instead.

Now that the min is 0, there isn't really a nonnegative val < 0 to check..

Reply:
I can remove the assertions on memory if that would make more sense

Reply:
removed a few where the min was not relevant